src/utils/utils.js

```diff
@@ -0,0 +1,52 @@
+// department
+const showDepartmentChoices = (departmentsFromDB) => {
+  // get the dept map them
+  return departmentsFromDB.map((department) => {
+    //return object key:value
+    return {
+      //key would be the name of dept and value is the id
+      name: department.dept_name,
```

Comment:
This does not return the department name because it is just `name` in the object.
```suggestion
      name: department.name,
```